openstack/neutron/templates/etc/_ssh_host_id_ec.tpl

```diff
@@ -1 +1 @@
-{{ .Values.sftp.server_key | required "Please set .Values.sftp.server_key" }}
+{{ .Values.sftp.server_key | required "Please set .Values.sftp.server_key" | include "resolve_secret" }}
```

Comment:
That shouldn't be needed, as the template is simply used in `openstack/neutron/templates/sftp-secrets.yaml` directly,
and the secret-injector should resolve those values automatically.

Reply:
Hmm, true, but it also doesn't hurt?
I would prefer to rather have a secret get piped through the injector twice but have all templates look consistent, which helps in reviewing and working with them, than miss one.
Also when later someone decides to include that template somewhere else in a different way, it will not get resolved.

Reply:
No, it doesn't hurt. Matter of taste.
I would prefer to make clear that actually there is no go template processing of the output happening there, as the string is replaced on server-side.